backend/canisters/identity/api/src/main.rs

```diff
@@ -8,6 +8,7 @@ fn main() {
     generate_candid_method!(identity, generate_challenge, update);
     generate_candid_method!(identity, migrate_legacy_principal, update);
     generate_candid_method!(identity, prepare_delegation, update);
+    generate_candid_method!(identity, set_principal_migration_job_enabled, update);
```

Comment:
I wondering why you haven't implemented the function? It seems like it would be very quick.

Reply:
It's already implemented. Just wasn't wired up on the frontend